fastparquet/test/test_util.py

```diff
@@ -5,25 +5,21 @@
 
 def test_analyse_paths():
     file_list = ['a', 'b']
-    base, out = analyse_paths(file_list, '/')
-    assert (base, out) == ('', ['a', 'b'])
+    base, out = analyse_paths(file_list)
+    assert (base, out) == ('.', ['a', 'b'])
```

Comment:
I hope that using `.` to represent a lack of path is okay.

Reply:
That's fine - I'm not immediately sure if this is used anywhere.